frontend/packages/data-portal/public/locales/en/translation.json

```diff
@@ -186,13 +191,15 @@
   "manual": "Manual",
   "meetsAll": "Meets all",
   "metadata": "Metadata",
+  "methodLinks": "Method Links",
   "methodType": "Method Type",
   "methodTypeAutomated": "<semibold>Automated Methods:</semibold> Any prediction that was not manually altered after being generated.",
   "methodTypeHybrid": "<semibold>Hybrid Methods:</semibold> Any annotation that has been curated/altered after generated with an automated method.",
   "methodTypeInfo": "<semibold>Method Type:</semibold> Field indicating what type of (if any) machine learning methods were involved in generating the annotation.",
   "methodTypeManual": "<semibold>Manual Methods:</semibold> Annotation was created with methods that are fully-supervised particle selection and segmentation.",
```

Comment:
[Update tooltip copy to reflect final copy from dannielle:](https://docs.google.com/spreadsheets/d/1QEPiDcjGJYWSZXsbRjwHd9aeZjbk_tvv2UQ4DKyMwxk/edit?gid=339544814#gid=339544814) 
`Manual: Annotations were created by hand`